app/default_prefs.py

```diff
@@ -153,6 +153,7 @@ def numberTest(str, expectRegs):
 
 color256 = {
   '_pre_selection': stringColorIndex,
+  'bookmarkColorIndexStart': 96,
```

Comment:
I added a check (assert) to look for an color256 and color8 getting out of sync. (so this should assert on startup unless this is added to color8 as well).

Reply:
Yep, 8-bit is supported now!

Reply:
Thanks!